src/operator/controllers/external_traffic/endpoints_reconciler.go

```diff
@@ -47,7 +49,7 @@ func (r *EndpointsReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&corev1.Endpoints{}).
-		WithOptions(controller.Options{RecoverPanic: true}).
+		WithOptions(controller.Options{RecoverPanic: lo.ToPtr(true)}).
 		Complete(r)
```

Comment:
All changes of `RecoverPanic` are a result of `sigs.k8s.io/controller-runtime` upgrade.